operations/services.md

```diff
@@ -186,9 +191,15 @@ Returns all availability blocks filtered by services, unique identifiers and oth
 | `CreatedUtc` | [Time interval](enterprises.md#time-interval) | optional, max length 3 months | Interval in which the [Availability block](#availability-block)s were created. |
 | `UpdatedUtc` | [Time interval](enterprises.md#time-interval) | optional, max length 3 months | Interval in which the [Availability block](#availability-block)s were updated. |
 | `CollidingUtc` | [Time interval](enterprises.md#time-interval) | optional, max length 3 months | Interval in which the [Availability block](#availability-block)s are active. |
+| `States` | array of string [Availability block state](#availability-block-state) | optional | States the availability blocks should be in. |
```

Comment:
we might want to specify that the confirmed is the default state?

Reply:
We want to get rid of the fallback before more partners will adopt it, so it'd not mention it.